lib/worker/index.js

```diff
@@ -31,14 +31,11 @@ const _ledgerNodeMeta = require('../temporaryLedgerNodeMeta');
 
 const {coolDownPeriod} = config['ledger-consensus-continuity'].gossip;
 
-api.scheduleWork = ({session}) => {
+api.scheduleWork = async ({session}) => {
   // start a consensus session for ledgers
-  const maxTime =
+  const maxAge =
```

Comment:
A comment about why the semantics are being changed here would be helpful.  I'm not even sure if maxAge/maxTime is clear, is this really `maxTtl`?